Bindings/bindings.json

```diff
@@ -1474,6 +1474,14 @@
                     ]
                 },
                 {
+                    "name": "apiKey",
+                    "value": "string",
+                    "defaultValue": "SendGridApiKey",
```

Comment:
Our existing setting name is AzureWebJobsSendGridApiKey, and that is what the extension is looking for, if not set in the binding, right? Should we change the name, or keep the existing name? For existing users who have this setting already, it'd be nice if the new template just worked.